docs/site/components/relative-link/relative-link.js

```diff
@@ -0,0 +1,17 @@
+(function(Showdown) {
+
+  'use strict';
+
+  Showdown.extensions.relativelink = function () {
```

Comment:
Can you move this to the home.js config block? Example: https://github.com/GoogleCloudPlatform/gcloud-node/pull/734#issuecomment-123827020